library/src/main/java/com/novoda/downloadmanager/Migration.java

```diff
@@ -21,6 +21,15 @@ Batch batch() {
         return fileMetadata;
     }
 
+    boolean hasDownloadedBatch() {
+        for (FileMetadata fileMetadatum : fileMetadata) {
```

Comment:
nice use of the singular `datum` here 👍

Reply:
I wish I could take credit here 😄  this was ASide 😄